engine/src/gfx/particle.cpp

```diff
@@ -294,16 +238,16 @@ void ParticleTrail::Config::init() {
     use = XMLSupport::parse_bool(vs_config->getVariable("graphics", prefix, "true"));
     use_points = XMLSupport::parse_bool(vs_config->getVariable("graphics", prefix + "point", "false"));
     pblend = XMLSupport::parse_bool(vs_config->getVariable("graphics", prefix + "blend", "false"));
-    pgrow = XMLSupport::parse_float(vs_config->getVariable("graphics",
+    pgrow = XMLSupport::parse_floatf(vs_config->getVariable("graphics",
```

Comment:
Not going to block on this; but as a long term goal I'd prefer the `float` aspect of this come from the config return and not have to get wrapped to parse it:
```
pgrow = vs_config->getFloatVariable("graphics",...)
```